test/output-tests/reparse.ml

```diff
@@ -0,0 +1,76 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                        SuperBOL OSS Studio                             *)
+(*                                                                        *)
+(*  Copyright (c) 2022-2023 OCamlPro SAS                                  *)
+(*                                                                        *)
+(* All rights reserved.                                                   *)
+(* This source code is licensed under the GNU Affero General Public       *)
+(* License version 3 found in the LICENSE.md file in the root directory   *)
+(* of this source tree.                                                   *)
+(*                                                                        *)
+(**************************************************************************)
+
+open Format
+open Ez_file
+open FileString.OP
+open Cobol_preproc
+
+let deep_iter = FileString.(make_select iter_dir) ~deep:true
+let srcdir = try Unix.getenv "DUNE_SOURCEROOT" with Not_found -> "."
```

Comment:
Take a look at https://github.com/OCamlPro/superbol-studio-oss/blob/2b9c53e13855c3cfaaf60b9d0cfab34cc9ea9207/test/output-tests/preproc.ml#L19 to make the relocation of tests ok

Reply:
I am not sure what the use for this is, can you clarify why/when tests are relocated?

Reply:
Ah but this is upstream changes to the file I copied, OK